source/TreeWalker.js

```diff
@@ -59,7 +59,7 @@ TreeWalker.prototype.nextNode = function () {
     }
 };
 
-TreeWalker.prototype.previousNode = function () {
```

Comment:
this is doing a reverse depth first traversal.  I needed to use it to replace the backspace key, basically when you have an element you need to find the element before it.  The way the walker works is that you give it a filter to where it will only match nodes fulfilling the filter.  So you can get only text nodes, or only non-text nodes, or both.  But with uneditable elements, I needed a way to stop the traversal if we ran upon one of them, rather than continuing down and finding their last ancestor, which would normally be a text element.

Thus the breakoutFunction.